plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/order-summary/style.scss

```diff
@@ -1,4 +1,7 @@
 .wc-block-components-order-summary {
+	// Compensate for removing Panel.
+	padding: 0 16px;
```

Comment:
Should we use our $gap sizes here instead of pixel values?

Reply:
whoops yep this is a mistake! thanks

Reply:
Resolved.